Jenkinsfile.kola.aws

```diff
@@ -83,6 +83,7 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
             aws_upgrade: {
                 stage('Kola:AWS upgrade') {
                     utils.shwrap("""
```

Comment:
Maybe later `utils.shwrap_aws()` or something like that?

Reply:
Yeah, makes sense. Hmm, or actually now that we're using the code-as-configuration plugin, we could do this the more Jenkins-y way by converting the Kube secrets to Jenkins credentials (like we do the Slack token now) and use `withCredentials() { ... }`. That'll make it easier to limit the creds exposed to just what the script needs.